packages/fcl-react-native/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@onflow/fcl-react-native",
+  "version": "1.8.1",
+  "description": "Flow Client Library",
+  "license": "Apache-2.0",
+  "author": "Dapper Labs <dev@dapperlabs.com>",
+  "homepage": "https://onflow.org",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/onflow/flow-js-sdk.git"
+  },
+  "bugs": {
+    "url": "https://github.com/onflow/flow-js-sdk/issues"
+  },
+  "jest": {
+    "testEnvironment": "jsdom",
+    "globals": {
+      "PACKAGE_CURRENT_VERSION": "TESTVERSION"
+    }
+  },
+  "devDependencies": {
+    "@onflow/fcl-bundle": "^1.4.1",
+    "@onflow/typedefs": "^1.2.1",
+    "@types/estree": "^1.0.1",
+    "@types/node": "^18.13.0",
+    "eslint": "^8.35.0",
+    "eslint-plugin-jsdoc": "^46.9.0",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "typescript": "^4.9.5"
+  },
+  "source": "src/fcl-react-native.ts",
+  "main": "dist/fcl-react-native.js",
+  "module": "dist/fcl-react-native.module.js",
+  "unpkg": "dist/fcl-react-native.umd.min.js",
+  "types": "types/fcl-react-native.d.ts",
+  "scripts": {
+    "pain": "npm publish --tag pain",
+    "alpha": "npm publish --tag alpha",
+    "prepublishOnly": "npm test && npm run build",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "build": "npm run lint && fcl-bundle",
+    "build:types": "tsc",
+    "start": "fcl-bundle --watch",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.18.6",
+    "@onflow/fcl-core": "1.8.1",
+    "@onflow/config": "^1.2.1",
+    "@onflow/interaction": "0.0.11",
+    "@onflow/rlp": "^1.2.1",
+    "@onflow/sdk": "^1.3.1",
+    "@onflow/types": "^1.2.1",
+    "@onflow/util-actor": "^1.3.1",
+    "@onflow/util-address": "^1.2.1",
+    "@onflow/util-invariant": "^1.2.1",
+    "@onflow/util-logger": "^1.3.1",
+    "@onflow/util-semver": "^1.0.1",
+    "@onflow/util-template": "^1.2.1",
+    "@onflow/util-uid": "^1.2.1",
+    "cross-fetch": "^3.1.6"
+  },
+  "peerDependencies": {
+    "@react-native-async-storage/async-storage": "^1.13.0",
+    "expo-linking": "^4.0.0",
+    "expo-web-browser": "^12.0.0",
+    "react": "^17.0.0 || ^18.0.0",
+    "react-native": "^0.0.0-0 || 0.60 - 0.72 || 1000.0.0"
+  }
```

Comment:
When we make follow up changes to fcl-react-native we can try to clean up these peer dependencies & see which ones we can transition to regular dependencies.